front/app/containers/Admin/reporting/components/ReportBuilder/Widgets/ChartWidgets/messages.ts

```diff
@@ -13,6 +13,11 @@ export default defineMessages({
     id: 'app.containers.admin.ReportBuilder.charts.analyticsChartDateRange',
     defaultMessage: 'Date range',
   },
+  adminInaccurateParticipantsWarning: {
+    id: 'app.containers.admin.ReportBuilder.charts.adminInaccurateParticipantsWarning',
+    defaultMessage:
+      'Please note that participation numbers may not be fully accurate as some data is captured in an external survey that we do not track',
```

Comment:
Should we add a period to the end of this sentence?

Reply:
Also - should we have "...in an external survey that we do not track" or "... in an externals survey that we cannot track"? :thinking:

Reply:
Gotcha! Ok sounds good :+1: